packages/frontend/src/components/accounts/create/verify_account/VerifyAccountWrapper.js

```diff
@@ -38,10 +39,17 @@ export function VerifyAccountWrapper() {
 
     useEffect(() => {
         const checkIfMoonPayIsAvailable = async () => {
-            const moonpayAvailable = await isMoonpayAvailable();
-            if (moonpayAvailable) {
-                setShowFundWithCreditCardOption(true);
-            }
+            await Mixpanel.withTracking("CA Check Moonpay available",
+                async () => {
+                    const moonpayAvailable = await isMoonpayAvailable();
+                    if (moonpayAvailable) {
+                        setShowFundWithCreditCardOption(true);
+                    }
+                },
+                (e) => {
+                    throw e;
+                }
```

Comment:
```suggestion
                }
```

re-throwing from the `errorOperation` with no other logic is equivalent to just not defining one at all :)